docs/kata-containers.md

```diff
@@ -0,0 +1,49 @@
+# Kata containers support in CRI-RM
+
+Kata containers represent an alternative runtime to Runc containers,
+and are conceptually defined by a containers which replace their sandbox 
+with a lightweight Virtual Machine.
```

Comment:
Here I'd also link to the main Kata site. I'd also steal the short summary of Kata/Kata Containers from their site, to avoid having to think too hard if the description is right. So something like this:

[Kata Containers](https://katacontainers.io/) is an open source container runtime, building lightweight virtual machines that seamlessly plug into the containers ecosystem.